ui/ui-components/pages/model/[modelId]/destinations.tsx

```diff
@@ -1,206 +0,0 @@
-import { useState } from "react";
-import Head from "next/head";
-import Link from "next/link";
-import { useRouter } from "next/router";
-import { Badge } from "react-bootstrap";
-import type { NextPageContext } from "next";
-import { UseApi } from "../../../hooks/useApi";
-import { useOffset, updateURLParams } from "../../../hooks/URLParams";
-import { useSecondaryEffect } from "../../../hooks/useSecondaryEffect";
-import EnterpriseLink from "../../../components/GrouparooLink";
-import Pagination from "../../../components/Pagination";
-import LoadingTable from "../../../components/LoadingTable";
-import AppIcon from "../../../components/AppIcon";
-import StateBadge from "../../../components/badges/StateBadge";
-import { Models, Actions } from "../../../utils/apiData";
-import { formatTimestamp } from "../../../utils/formatTimestamp";
-import { ErrorHandler } from "../../../utils/errorHandler";
-import LinkButton from "../../../components/LinkButton";
-import { grouparooUiEdition } from "../../../utils/uiEdition";
-import { formatName } from "../../../utils/formatName";
-import DestinationCollectionLink from "../../../components/destination/DestinationCollectionLink";
```

Comment:
I wonder if this component can be removed

Reply:
I reviewed a lot of these components. I created this component to share between the model overview and this page. But now that the page is removed the overview still uses it